test/pom.xml

```diff
@@ -182,6 +182,12 @@ THE SOFTWARE.
       <version>4.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>cloudbees-folder</artifactId>
+      <version>6.0.4</version>
```

Comment:
What circular dependency? I see no dependency on `org.jenkins-ci.main:test` in `cloudbees-folder`?

The same could then be said about `antisamy-markup-formatter`, `junit` or `structs`.